servers/src/main/java/tachyon/master/block/BlockMaster.java

```diff
@@ -345,8 +351,11 @@ public void commitBlock(long workerId, long usedBytesOnTier, String tierAlias, l
         if (masterBlockInfo == null) {
           masterBlockInfo = new MasterBlockInfo(blockId, length);
           mBlocks.put(blockId, masterBlockInfo);
-          writeJournalEntry(
-              new BlockInfoEntry(masterBlockInfo.getBlockId(), masterBlockInfo.getLength()));
+          BlockInfoEntry blockInfo = BlockInfoEntry.newBuilder()
+              .setBlockId(masterBlockInfo.getBlockId())
+              .setLength(masterBlockInfo.getLength())
+              .build();
+          writeJournalEntry(JournalEntry.newBuilder().setBlockInfo(blockInfo).build());
```

Comment:
Alternatively, instead of overloading `writeEntry`, you can add a method `wrap` in `JournalProtoUtils` which converts each Journal entry sub type in to a `JournalEntry` (basically revert `unwrap`) and call `writeJournalEntry` with wrapped `JournalEntry`? In this way, we still keep the boilerplate code minimum and centralized.